src/lidar/read_rp_lidar.cpp

```diff
@@ -12,61 +12,84 @@
 
 using namespace navtypes;
 
+constexpr uint32_t RPLIDAR_BAUDRATE = 115200;
+const double MM_TO_M = 1000;
+
 std::atomic<bool> lidar_initialized(false);
-RPLidar rp_lidar("/dev/ttyUSB0");
+std::atomic<datatime_t> lidar_time(datatime_t{});
+RPLidar rp_lidar("/dev/ttyUSB0", RPLIDAR_BAUDRATE);
```

Comment:
We should probably store `"/dev/ttyUSB0"` in a constant in `Constants.h` as well.